modules/mobile/app/helpers/mobile/facilities_helper.rb

```diff
@@ -36,7 +41,8 @@ def address_from_facility(facility)
         zip_code = address[:postal_code]
       else
         address = facility.address['physical']
-        street = address.slice('address_1', 'address_2', 'address_3').values.compact.join(', ')
+        street = address.slice('address1', 'address2', 'address3', 'address_1', 'address_2',
+                               'address_3').values.compact.join(', ')
```

Comment:
👍 The v0 API has these `snake_cased` while v1 has the underscores removed with the shift to `camelCase`. Conveniently, when we `compact` the list the half of these that are `nil` get dropped, so we wind up with a join that covers both sides of the feature flag.

Still better to get rid of the old ones when we drop the flag, of course.